modules/toast/react/lib/Toast.tsx

```diff
@@ -52,7 +52,7 @@ const Message = styled('div')({
 
 export default class Toast extends React.Component<ToastProps> {
   static defaultProps = {
-    icon: checkIcon,
+    icon: checkIcon as CanvasSystemIcon, // needed for TS2742 - https://github.com/microsoft/TypeScript/issues/29808
```

Comment:
There might now be a type error with the latest version of `@workday/design-assets-types`. It was bumped from `0.2.4` to `0.2.6`. There is an open issue with no resolution for when it happens. It is not fully understood why.